test/cli_volume_test.go

```diff
@@ -137,3 +137,16 @@ func (suite *PouchVolumeSuite) TestVolumeCreateWithLabel(c *check.C) {
 	command.PouchRun("volume", "create", "--name", funcname, "--label", "test=foo").Assert(c, icmd.Success)
 	command.PouchRun("volume", "remove", funcname)
 }
+
+// TestVolumeCreateWithSelector tests creating volume with --selector.
+func (suite *PouchVolumeSuite) TestVolumeCreateWithSelector(c *check.C) {
+	pc, _, _, _ := runtime.Caller(0)
+	tmpname := strings.Split(runtime.FuncForPC(pc).Name(), ".")
+	var funcname string
+	for i := range tmpname {
+		funcname = tmpname[i]
+	}
+
+	command.PouchRun("volume", "create", "--name", funcname, "--selector", "test=foo").Assert(c, icmd.Success)
```

Comment:
Could we test getting this selector via `pouch volume inspect`?
Otherwise we cannot test whether this setting works in the daemon side. Right?

Reply:
pouch volume inspect doesn't show the selector info currently:
```
#pouch volume create --selector test=foo --name test20
Mountpoint:
Name:         test20
Scope:
CreatedAt:
Driver:       local

#pouch volume inspect test20
CreatedAt:    2018-2-7 11:05:41
Driver:       local
Mountpoint:   /mnt/local/test20
Name:         test20
Scope:

```

Reply:
Got it